auth/auth-router.js

```diff
@@ -0,0 +1,70 @@
+const express = require('express');
+const bcrypt = require('bcryptjs');
+const jwt = require('jsonwebtoken');
+const usersModel = require('../users/users-model');
+const secrets = require('../config/secrets');
+
+const router = express.Router();
+
+router.post('/register', async (req, res, next) => {
+  try {
+    const user = await usersModel.add(req.body);
```

Comment:
Nothing wrong with directly adding the req.body, but I would really like to see you whitelist the needed fields